tekton/ci-lint.yaml

```diff
@@ -1,33 +1,71 @@
 apiVersion: tekton.dev/v1alpha1
 kind: Task
 metadata:
-  name: golangci-lint
+  name: lint
 spec:
   inputs:
     params:
     - name: package
       description: package (and its children) under test
       default: github.com/tektoncd/pipeline
-    - name: flags
+    - name: golangci-flags
       description: flags to use for the lint command
       default: --verbose
-    - name: version
+    - name: golangci-version
       description: golangci-lint version to use
       default: v1.16
     resources:
     - name: source
       type: git
       targetPath: src/${inputs.params.package}
   steps:
-  - name: lint
-    image: golangci/golangci-lint:${inputs.params.version}
+  - name: golangci-lint
+    image: golangci/golangci-lint:${inputs.params.golangci-version}
     workingdir: /workspace/src/${inputs.params.package}
     command:
     - /bin/bash
     args:
     - -c
-    - "golangci-lint run ${inputs.params.flags}"
+    - "golangci-lint run ${inputs.params.golangci-flags}"
     env:
     - name: GOPATH
       value: /workspace
+  - name: header-lint
+    image: bash
+    workingdir: /workspace/src/${inputs.params.package}
+    command:
+      - bash
+    args:
+      - -ce
+      - |
+        header='/*
+        Copyright 2019 The Tektocmd/entrypoint/main.go Authors
```

Comment:
Uhm, this will force the header to 2019 always, which I'm not sure it's what we want.

Reply:
Thanks for your comment! I tried using regex instead such as `Copyright 20[0-9]{2} The Tekton Authors` but it does not seem to work correctly. Would you have any good ideas how to match with other years?